Examples/ProductContent.cs

```diff
@@ -28,14 +31,20 @@ public class ProductContent
         public static async Task<FileDownloadModel> GetAttachment(
             IReportsLibrary iReportsLibrary, string productCode, string fileCode)
         {
-            FileDownloadModel attachmentModel = await iReportsLibrary.GetAttachmentAsync(
+            FileDownloadModel attachmentModel = await iReportsLibrary.GetAttachmentAsync(           
                 "reports", productCode, fileCode);
 
             Console.WriteLine($"File name = {attachmentModel.Name}");
 
             return attachmentModel;
         }
 
+        public static void SaveAttachementToWorkingDirectory(FileDownloadModel fileModel){
+            FileStream fileStream = File.Create($"{Directory.GetCurrentDirectory()}/{fileModel.Name}");
```

Comment:
I think that you can just do File.WriteAllBytes("filename", fileModel.Content);
Also you can just use the file name and it will automatically put it in the current directory :)